autoload/nvimdev.vim

```diff
@@ -145,9 +147,6 @@ function! nvimdev#init(path) abort
   augroup nvimdev
     autocmd!
     autocmd BufRead,BufNewFile *.h set filetype=c
-    if get(g:, 'nvimdev_auto_lint', 1)
```

Comment:
You mean disable the clint maker when running `:Neomake`?  I suppose an entirely different option.  It can still be ran with `:Neomake lint`.  But, I don't see how this would be beneficial.

If you're talking about the `autocmd` itself, it needs to be fixed so it's default is `0`.  The problem you originally mentioned would then be resolved.